lib/components/common/StyledCard.tsx

```diff
@@ -31,6 +31,7 @@ const StyledCard: React.FC<StyledCardProps> = ({value, isSuggested, selectedValu
         if (value !== selectedValue) {
             onChange(value);
         }
+        onAfterChange();
```

Comment:
question: Why do we need this one? Can't we just use the onChange?

Reply:
@afonsobspinto I tried to update tour step just in onChange function, but it doesn't work, that is why we need this onAfterChange function

Reply:
Would it be possible for you to elaborate a bit on why it doesn't work?

Reply:
This question is still pending @Aiga115

Reply:
`const handleRadioChange = () => {
        if (value !== selectedValue) {
            onChange(value);
        }
        // onAfterChange();
    `};``
    because of this condition and because from the start of application we have already selected value, onChange function just doesn't get triggered, that is why I put onAfterChange here, so even if we don't update to new value we still trigger the next step @afonsobspinto

Reply:
I understand. I believe you can still merge the two and move the check for real changes to the parent component. If you prefer to keep both I would make the case to have a better naming for the prop. I don't think it makes much sense to call a method onAfterChange if it's called even if nothing changes, do you agree?